.github/workflows/jobs.yml

```diff
@@ -1,11 +1,11 @@
-name: "Terraform"
+name: "Release"
```

Comment:
Should we call this something more generic since this will run on more than just releases?

Reply:
If it's manually run for release, it will be called `Release`, otherwise it's the PR name. See https://github.com/quay/mirror-registry/actions